lua/telescope/_extensions/luasnip.lua

```diff
@@ -1,154 +1,185 @@
+    separator = ' ',
+    items = { { width = 12 }, { width = 24 }, { width = 16 }, { remaining = true } },
+  })
+
+  local make_display = function(entry)
+    return displayer({
+      entry.value.ft,
+      entry.value.context.name,
+      { entry.value.context.trigger, 'TelescopeResultsNumber' },
+      filter_description(entry.value.context.name, entry.value.context.description),
     })
-
-    local make_display = function(entry)
-        return displayer({
-            entry.value.ft,
-            entry.value.context.name,
-            { entry.value.context.trigger, "TelescopeResultsNumber" },
-            filter_description(entry.value.context.name, entry.value.context.description),
-        })
-    end
-
-    -- stylua: ignore
-    pickers.new(opts, {
-        prompt_title = "LuaSnip",
-        finder = finders.new_table({
-
-            results = objs,
-            entry_maker = function(entry)
-		    search_fn = ext_conf._config.luasnip and ext_conf._config.luasnip.search or default_search_text
-                return {
-                    value = entry,
-                    display = make_display,
-
-		    ordinal = search_fn(entry),
-
-                    preview_command = function(_, bufnr)
-                        local snippet = get_docstring(luasnip, entry.ft, entry.context)
-                        vim.api.nvim_buf_set_option(bufnr, "filetype", entry.ft)
-                        vim.api.nvim_buf_set_lines(bufnr, 0, -1, false, snippet)
-                    end,
-                }
-            end,
-        }),
-
-        previewer = previewers.display_content.new(opts),
-        sorter = conf.generic_sorter(opts),
-        attach_mappings = function()
-            actions.select_default:replace(function(prompt_bufnr)
-                local selection = action_state.get_selected_entry()
-                actions.close(prompt_bufnr)
-                vim.cmd('startinsert')
-                vim.api.nvim_put({ selection.value.context.trigger }, "", true, true)
-                if (luasnip.expandable()) then
-                    luasnip.expand()
-                else
-                    print("Snippet '" .. selection.value.context.name .. "'" .. "was selected, but LuaSnip.expandable() returned false")
-                end
-                vim.cmd('stopinsert')
-            end)
-            return true
-        end,
-    }):find()
+  end
+
+	-- stylua: ignore
+	pickers.new(opts, {
+		prompt_title = "LuaSnip",
+		finder = finders.new_table({
+			results = objs,
+			entry_maker = function(entry)
+				search_fn = ext_conf._config.luasnip
+					and ext_conf._config.luasnip.search
+					or default_search_text
+				return {
+					value = entry,
+					display = make_display,
+					ordinal = search_fn(entry),
+					preview_command = function(_, bufnr)
+						local snippet = get_docstring(luasnip, entry.ft, entry.context)
+						if opts.preview.check_mime_type then
+							vim.api.nvim_buf_set_option(bufnr, "filetype", entry.ft)
+						end
```

Comment:
Here you go!

Reply:
?Cheers. I'm seeing some stuff formatted with tabs though. Not sure if that's intentional?